install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -15,6 +15,9 @@ resources:
   requests:
     cpu: 500m
     memory: 2048Mi
+  limits:
+    cpu: 4800m
+    memory: 4G
```

Comment:
Where is 4gb from? In our large scale tests we use 12gb. 

I want to avoid just adding limits for the sake of adding limits. Its not a great UX for pilot to shut down due to OOM just because you added 1 config too many and hit the limit. Users can easily set the limit in helm